psydac/api/essential_bc.py

```diff
@@ -89,10 +104,10 @@ def apply_essential_bc_stencil(a, *, axis, ext, order, identity=False):
     Parameters
     ----------
     a : StencilVector, StencilMatrix or StencilInterfaceMatrix
-        The Matrix or the Vector that will be modified.
+        The matrix or the Vector to be modified.
```

Comment:
'Vector' is still with capital letter

Reply:
I think that this was intentional, as `Vector` is the name of the type while matrix is not...